File: JavaScript/프로그래머스/1/12982. 예산/예산.js
function solution(d, budget) {
    let answer = 0;
    let count = 0
    d.sort((a,b)=>a-b)
    for(let i = 0 ; i < d.length ; i++){
        answer += d[i]
        count++
        if(budget-answer < 0){
            answer -= d[i]
            count--
            break
        }
    }
    return count
}

File: Python/프로그래머스/1/12982. 예산/예산.py
def solution(d, budget):
    answer = 0
    count = 0
    d.sort()
    
    for i in range(len(d)) :
        answer += d[i]
        count+=1
        if budget-answer < 0 :
            answer -= d[i]
            count-=1
            break
        
    return count